src/Explorer/QueryCopilot/QueryCopilotUtilities.ts

```diff
@@ -31,15 +40,22 @@ export const submitFeedback = async (params: FeedbackParams): Promise<void> => {
       description: description || "",
       contact: contact || "",
     };
-
-    await fetch("https://copilotorchestrater.azurewebsites.net/feedback", {
+    if (shouldAllocateContainer) {
+      await explorer.allocateContainer();
+      setShouldAllocateContainer(false);
+    }
+    const serverInfo = useNotebook.getState().notebookServerInfo;
+    const response = await fetch(`${serverInfo.notebookServerEndpoint}feedback`, {
```

Comment:
Use createUri() from src\Common\UrlUtility to create the url instead of concatenating.

Reply:
Changed the code.